doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md

```diff
@@ -72,7 +72,7 @@ The GCC flag `-latomic` is not being added automatically with specific architect
   * If no formats have working thumbnails, then it is an issue with the `f3d-plugin-xxx.thumbnailer` files.
   * If only big file do not have thumbnails, this is intended, you can modify this behavior in the `thumbnail.d/10_global.json` configuration folder using the `max-size` option.
 
-Some file managers (eg: Nautilus) are using sandboxing for thumbnails, which the F3D standard binary release does not support as it needs
+
 access to the Xorg server to render anything.
 In that case, you want to use the headless version of F3D that rely on EGL instead of Xorg, available in the [releases page](https://github.com/f3d-app/f3d/releases).
```

Comment:
this whole part should be gone

Reply:
@mwestphal the whole thumbnail not working part??